### DISCUSSION TITLE
Can we use cameras like Iriun Webcam?

### DISCUSSION THREAD

**Discussion by JuanmaWL** [hacksider/Deep-Live-Cam#93] [General]
Hello there👋🏻

🔧 Issue:

I have everything set up, but when I click Live, the preview shows a "VCAM" blue logo in the middle.

📷 Context:

The only "webcam" I have is my mobile phone running a WiFi camera app (Iriun Webcam) to use the phone's camera as a webcam for the computer.

❓ Question:
How can I switch to this webcam instead of the virtual camera in the preview? I need to apply the deepfake to my actual webcam.
Do I need to make any changes in Windows 11 or use OBS? I'm not sure if these are related.
![111](https://github.com/user-attachments/assets/1b4f8d47-099e-4be9-8188-42f7aad7d353)

Thank you in advance!

**Comment by JuanmaWL** [comment]
Answering my own question because I was being completely clueless, haha. I had a virtual camera installed called "Animaze Virtual Camera" that didn't uninstall properly. You need to remove it from Device Manager, and then Iriun Webcam gets recognized as the primary camera. Problem solved, lol!

**Comment by Enchante503** [comment]
Please tell us about your environment.
I'm using Windows WSL2 and a smartphone camera,
but it's not working.

**Reply by JuanmaWL** [reply]
In my case, I'm using native Windows 11 and the Iriun Webcam software (both on Windows and an Android phone). The phone is a Xiaomi 9T Pro with MIUI, but I've also tested it on a Meizu M6 with a custom ROM (Lineage OS). In the Windows settings, only the Iriun Webcam camera appears, exclusively. My previous problem was that I had a leftover webcam from another program that I had left uninstalled.

![image](https://github.com/user-attachments/assets/bc97f695-c6e7-4a8d-840a-d75b744df2bb)

**Comment by ffpaniago** [comment]
On the ui.py you have the option to select the webcam.

The default value is 0 to the main camera; I use a GoPro as cam, than I set this value to 1

**ui.py**
`cap = cv2.VideoCapture(1)  # Use index for the webcam (adjust the index accordingly if necessary)  `
